README.md

```diff
@@ -215,87 +215,184 @@ $('.submit-sms-code').click(function (ev) {
 });
 ```
 
-### Processing the callback
+### Processing the Callback
 
-### Redirect Mode
+How does control return back to your app after a login has been attempted?  This all depends on which login "mode" you choose to use (**Redirect** or **Popup**) and in some cases, which type of connection you're using.
 
-Once you have succesfully authenticated, Auth0 will redirect to your `callbackURL` with a hash containing an `access_token` and the jwt (`id_token`). You can parse the hash and retrieve the full user profile as follows:
+#### Redirect Mode
+
+The default mode of the `login` method is Redirect Mode.  Here two separate "redirect" actions will occur when `login` is called.  First, the browser will navigate to a separate login page to collect the user's credentials.  Once the user successfully logs in, the browser will redirect the user *back* to your application via the `callbackURL`.
+
+For example, let's say you've initialized your Auth0 client as shown in the [Initialize](#initialize) section above.  Then the following call to `login` using your `google-oauth2` social connection would result in a redirect to a Google login page and then a redirect back to `http://my-app.com/callback` if successful:
 
 ```js
-  $(function () {
-    var result = auth0.parseHash(window.location.hash);
+auth0.login({
+  connection: 'google-oauth2'
+});
```

Comment:
shouldn't we add to the example the `callbackURL: 'http://my-app.com/callback'` so it follows the example described above?